hawkular-inventory-parent/hawkular-inventory-itest/src/test/java/org/hawkular/inventory/handlers/InventoryRestTest.java

```diff
@@ -647,7 +647,7 @@ public void test105_shouldCreateAPrometheusJsonConfig() {
                     .id(id + "-" + i)
                     .feedId(feedId + "-" + i)
                     .typeId(type)
-                    .config("Metrics Endpoint", "localhost:1234")
+                    .config("Metrics Endpoints", "localhost:1234")
```

Comment:
Could you add a test with several endpoints?

Reply:
test added: https://github.com/hawkular/hawkular-commons/pull/202/commits/b79845d637bda6d76301a24afe17b9f09a46eca2#diff-a45ec6ef6e64c4766731c5e1d39d92c9R408